projects/plugins/crm/includes/ZeroBSCRM.InternalAutomatorRecipes.php

```diff
@@ -1297,42 +1300,78 @@ function zeroBSCRM_IA_NewCustomerWPHook($obj=array()){
 		}
 
 	}
-   	#} Fires on 'customer.edit' IA 
-	function zeroBSCRM_IA_EditCustomerWPHook($obj=array()){
 
-		if (is_array($obj) && isset($obj['id']) && !empty($obj['id'])) {
-		
-			do_action( 'jpcrm_after_contact_update', $obj['id'] );
+/**
+ * Fires on 'contact.update', 'contact.email.update', and 'contact.status.update IA.
+ *
+ * @param array $obj An array holding contact object data.
+ */
+function zeroBSCRM_IA_EditCustomerWPHook( $obj = array() ) {
+
+	if ( is_array( $obj ) && isset( $obj['id'] ) && ! empty( $obj['id'] ) ) {
+
+		do_action( 'jpcrm_after_contact_update', $obj['id'] );
 
-			// legacy, use `jpcrm_after_contact_update` from 5.3+
-			do_action( 'zbs_edit_customer', $obj['id'] );
+		// legacy, use `jpcrm_after_contact_update` from 5.3+
+		do_action( 'zbs_edit_customer', $obj['id'] );
 
+		// If the contact status has changed.
+		if ( isset( $obj['from'] ) && ! empty( $obj['from'] ) ) {
+			do_action( 'jpcrm_automation_contact_status_update', $obj );
+		} elseif ( isset( $obj['userMeta']['zbsc_email'] ) && $obj['prev_contact']['email'] !== $obj['userMeta']['zbsc_email'] ) {  // If the contact email has changed.
+			do_action( 'jpcrm_automation_contact_email_update', $obj );
+		} else { // Fallback if any other contact field has changed.
+			do_action( 'jpcrm_automation_contact_update', $obj );
```

Comment:
They should be parallel conditions, we have to check the `status`, the `email`, and the `fallback` case. For example, a contact update could be status and email change, so we should trigger both events.

Reply:
Ah got it, thanks! Added another commit to change that.